R/cols-best_case_worst_case.R

```diff
@@ -0,0 +1,43 @@
+col_companies_id <- function() {
+  "companies_id"
+}
+
+col_ep_product <- function() {
+  "ep_product"
+}
```

Comment:
This seems like a good opportunity to spell out "ep" as in "europages" and avoid the confusion with "ep" as in "emissions-profile".

I propose:

```r
col_eurpages_product <- function() "ep_product"
```